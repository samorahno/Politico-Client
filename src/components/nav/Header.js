import React from 'react';
import '../../styles/landing.css';
const Header = () => {
    return (
        <div>
            <div className="header1">
                <a href="index.html">Politico</a>
            </div>
        </div>
    );
};

export default Header;
