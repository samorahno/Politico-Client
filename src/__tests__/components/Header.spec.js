import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/nav/Footer';

describe('Header', () => {
    it('should render header component correctly', () => {
        const component = shallow(<Header />);
        expect(component).toMatchSnapshot();
    });
});
