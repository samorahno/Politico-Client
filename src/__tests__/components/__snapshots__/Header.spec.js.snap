// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render header component correctly 1`] = `ShallowWrapper {}`;
